webapp/app/statisticalFactsheets/components/levelSelection/levelSelection.js

```diff
@@ -0,0 +1,62 @@
+import './levelSelection.less'
+
+import React, { useEffect, useRef, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { useParams } from 'react-router'
+
+import Icon from '@webapp/components/icon'
+import useI18n from '@webapp/components/hooks/useI18n'
+import useGetRequest from '@webapp/components/hooks/useGetRequest'
+import * as CountryState from '@webapp/app/country/countryState'
+
+import List from './list'
+
+const levels = {
+  global: 'WO',
+  region: ['AF', 'AS', 'EU', 'NA', 'OC', 'SA'],
+}
+
+const LevelSelection = () => {
+  const i18n = useI18n()
+  const { levelIso } = useParams()
+
+  const { data = [], dispatch: fetchLevelIso } = useGetRequest('/api/statisticalFactsheets/levelIso')
+
+  const countrySelectionRef = useRef(null)
+  const [open, setOpen] = useState(false)
+
+  const outsideClick = (evt) => {
+    if (!countrySelectionRef.current.contains(evt.target)) setOpen(false)
+  }
+
+  useEffect(() => {
+    fetchLevelIso()
+    window.addEventListener('click', outsideClick)
+
+    return () => {
+      window.removeEventListener('click', outsideClick)
+    }
+  }, [])
+
+  const getLabel = () => {
```

Comment:
better to have a constant. no need for function.
also, do u run in a hooks change order warning?